hack/tools/go.sum

```diff
@@ -0,0 +1,585 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
not sure if you did, but you'll want to `go mod tidy`,  it also minimizes the sum file a bit

Reply:
done